games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit eutils cmake-utils gnome2-utils vcs-snapshot user
+
+DESCRIPTION="An InfiniMiner/Minecraft inspired game"
+HOMEPAGE="http://minetest.net/"
+SRC_URI="http://github.com/minetest/minetest/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+ CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+curl dedicated doc leveldb luajit nls redis +server +sound +truetype"
+
+RDEPEND="dev-db/sqlite:3
+	sys-libs/zlib
+	curl? ( net-misc/curl )
+	!dedicated? (
+		app-arch/bzip2
+		>=dev-games/irrlicht-1.8-r2
+		media-libs/libpng:0
+		virtual/jpeg:0
+		virtual/opengl
+		x11-libs/libX11
+		x11-libs/libXxf86vm
+		sound? (
+			media-libs/libogg
+			media-libs/libvorbis
+			media-libs/openal
+		)
+		truetype? ( media-libs/freetype:2 )
+	)
+	leveldb? ( dev-libs/leveldb )
+	luajit? ( dev-lang/luajit:2 )
+	nls? ( virtual/libintl )
+	redis? ( dev-libs/hiredis )"
+DEPEND="${RDEPEND}
+	>=dev-games/irrlicht-1.8-r2
+	doc? ( app-doc/doxygen media-gfx/graphviz )
+	nls? ( sys-devel/gettext )"
+
+pkg_setup() {
+	if use server || use dedicated ; then
+		enewgroup ${PN}
+		enewuser ${PN} -1 -1 /var/lib/${PN} ${PN}
+	fi
+}
+
+src_unpack() {
+	vcs-snapshot_src_unpack
```

Comment:
Tip: explicit `src_unpack` is unnecessary here, since `vcs-snapshot` is the last eclass exporting `src_unpack()` on inherit line. However, you can leave it for clearness if you like.